arch/arm/src/sama5/sam_adc.c

```diff
@@ -1728,7 +1890,8 @@ static void sam_adc_gain(struct sam_adc_s *priv)
   /* Set GAIN0 only.  GAIN0 will be used for all channels. */
 
   sam_adc_putreg(priv, SAM_ADC_CGR, ADC_CGR_GAIN0(CONFIG_SAMA5_ADC_GAIN));
-#endif
+#endif /* CONFIG_SAMA5_ADC_ANARCH */
```

Comment:
```suggestion
#  endif /* CONFIG_SAMA5_ADC_ANARCH */
```

Reply:
Can't find that set out in the Coding Style guide? Have I missed it?

Reply:
Ah, OK - yes, I see the associated #ifdef is (correctly) indented. Makes sense. Sorry!

Done.